project/Deployment.scala

```diff
@@ -128,10 +128,11 @@ class Deployment(
           |  -Dconfig.file=/home/$userName/scaladex-credentials/application.conf \\
           |  -Dsentry.dsn=$sentryDsn \\
           |  -Dsentry.release=$version \\
-          |  -Dapi.port=$port \\
-          |  /home/$userName/scaladex-contrib \\
-          |  /home/$userName/scaladex-index \\
-          |  /home/$userName/scaladex-credentials \\
+          |  -Dapp.port=$port \\
+          |  -Dapp.env=prod \\
+          |  -Ddata-paths.contrib=/home/$userName/scaladex-contrib \\
+          |  -Ddata-paths.index=/home/$userName/scaladex-index \\
+          |  -Ddata-paths.credentials=/home/$userName/scaladex-credentials \\
```

Comment:
Can we remove that and put them in `~/scaladex-credentials/app.conf` ?

Reply:
yes! next step . just to make this release not dependent on anything